### PR TITLE
Adds PlaceholderSvg tag

### PR DESCRIPTION
**Pull request by elseloop**
## Description
Adds a new PlaceholderSvg tag for building placeholder/FPO images using an SVG.

## Motivation and Context
[FPO Image](http://fpoimg.com/) or [Fill Murray](http://www.fillmurray.com/) only get us so far and can be slow. This provides an easy and composable alternative by building an inline SVG with a bunch of display options.

## How Has This Been Tested?
I used it. I wrote a simple test. That's about it.

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: source/tags/PlaceholderSvg/PlaceholderSvg.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    fontFamily,
+    fontSize,
+    fontWeight
+  } = props;
+
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox={`0 0 ${width} ${height}`}
+      width='100%'
```
Comment (krambuhl) [inline source/tags/PlaceholderSvg/PlaceholderSvg.jsx]: mixed single and double quotes.  please use double for html attributes and single for js properties.

File: source/tags/PlaceholderSvg/PlaceholderSvg.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      style={{ maxWidth: '100%'}}
```
Comment (krambuhl) [inline source/tags/PlaceholderSvg/PlaceholderSvg.jsx]: single quotes are sweet here.

File: source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PlaceholderSvg from './PlaceholderSvg';
+
+export default [{
+  name: "default",
+  component: (
+    <PlaceholderSvg width={600} height='300' />
```
Comment (krambuhl) [inline source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx]: quotes 🐺

**Comment by krambuhl** [comment]
@elseloop This is a good add.  A few code style tweaks above and then a question.

It looks like it always fills to the container width, is that expected?  I would expect `width` and `height` properties to have an effect over the max size, but it appears to really control the aspect ratio. Can you try to make this more clear, either by implementing `width`/`height` as `max-width`/`max-height` or by replacing `width`/`height` with a key like `aspectRatio`?

File: source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx (hunk 3 above)
Comment (elseloop) [inline source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx]: @krambuhl Sorry, which is which here? `height` and `width` are `props`, not HTML attributes, so shouldn't that be a single quote? And `name` is in the `new-tag` template with double quotes, so I'm guessing you don't mean that? My quote-blindness is getting the better of me here!

**Comment by elseloop** [comment]
@krambuhl I pushed some changes that address your notes:

* made the `height` and `width` props actually, you know, set the height and width of the `svg` 😬 
* reorganized the jsx file to better accommodate the new, longer styles block (but keeping them inline so they're not compiled and included in a delivered stylesheet)
* added a second test to show off all available props
* updated the readme to explicitly indicate that `height` and `width` should be unitless values

File: source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx (hunk 3 above)
Comment (elseloop) [inline source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx]: @krambuhl I left this as is and pushed what I had updated. I'll fix it the quote blizzard here on a later commit once I'm clearer on what needs to happen.

Comment (krambuhl) [inline source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx]: @elseloop you are correct about the new-tag template, that is an artifact of starting those files as JSON files, where double quotes are expected.

**Comment by elseloop** [comment]
@krambuhl quotation marks normalized! I left the `name` values in test wrapped in doubles because that's how they are everywhere right now so didn't want to create further quotation confusion.

**Comment by krambuhl** [comment]
🎷  that is some sweet music

**Pull request merged**